### PR TITLE
New methods for Input class

### PR DESCRIPTION
**Pull request by AlbertoBittes**
# Description

Two new methods (with tests):
- add material (and density) to void cells.
- add fill=/u= parameter keyword to cells that don't have it.

Additional dependencies introduced: None

## Type of change

Please select what type of change this is.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature 
    - [x] Non-breaking change which adds functionality
    - [ ] Breaking change fix or feature that would cause existing functionality to not work as expected

## Other changes

- [ ] This change requires a documentation update

# Testing

- test_add_material
- test_add_cell_fill_u

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] General testing 
    - [ ] New and existing unit tests pass locally with my changes
    - [ ] Coverage is >80%

### CODE REVIEW

File: src/f4enix/input/MCNPinput.py
Comment (dodu94) [inline src/f4enix/input/MCNPinput.py:1]: Can you please implement this add_material_to_void() in the replace method?

https://github.com/Fusion4Energy/F4Enix/blob/813dfcb6550451bfe81a643c43db8cc6471a51c0/src/f4enix/input/MCNPinput.py#L1118

at the moment it raises a ``NotImplementedError`` if one tries to substitute a void cell.

Also please check the doc of add_cell_fill_u for the definition of the cell argument (copy paste).

**Review by dodu94 — COMMENTED** [review]
Thanks Alberto great job!

A small request,
Davide

File: src/f4enix/input/MCNPinput.py
Comment (AlbertoBittes) [reply src/f4enix/input/MCNPinput.py:1]: Solved, with tests added

**Review by dodu94 — APPROVED** [review]

File: src/f4enix/input/MCNPinput.py
Comment (dodu94) [reply src/f4enix/input/MCNPinput.py:1]: thanks!

**Pull request merged**